homeassistant/components/husqvarna_automower/entity.py

```diff
@@ -120,3 +153,24 @@ class AutomowerControlEntity(AutomowerAvailableEntity):
     def available(self) -> bool:
         """Return True if the device is available."""
         return super().available and _check_error_free(self.mower_attributes)
+
+
+class WorkAreaControlEntity(AutomowerControlEntity):
+    """Replies available when the mower is connected."""
+
+    def __init__(
+        self,
+        mower_id: str,
+        coordinator: AutomowerDataUpdateCoordinator,
+        work_area_id: int,
+    ) -> None:
+        """Initialize AutomowerEntity."""
+        super().__init__(mower_id, coordinator)
+        self.work_area_id = work_area_id
+
+    @property
+    def work_area_attributes(self) -> WorkArea:
+        """Get the mower attributes of the current mower."""
```

Comment:
Stale docstring

Reply:
done in https://github.com/home-assistant/core/pull/126376/commits/7511f4dcfe2a815c2dfc42d8509c9853d310112c